app/components/header.tsx

```diff
@@ -1,11 +1,77 @@
-import { Text, Center } from '@chakra-ui/react';
+import {
+  Text,
+  Heading,
+  Avatar,
+  Flex,
+  HStack,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  IconButton,
+} from '@chakra-ui/react';
+import { TriangleUpIcon, LockIcon } from '@chakra-ui/icons';
+import { Link, useFetcher } from '@remix-run/react';
+
+import { useUser } from '~/utils';
 
 export default function Header() {
+  const user = useUser();
+  const fetcher = useFetcher();
   return (
     <header>
```

Comment:
You can remove `header` and instead just add property `as="header"` for Flex which is a child of `header`. It gonna make Flex render as `header`

Reply:
Also, I think it would be good to wrap header's content in `Container` too, so it's content would align with content of pages

Reply:
> You can remove header and instead just add property as="header" for Flex which is a child of header. It gonna make Flex render as header

oh didn't know I can do something like that

> Also, I think it would be good to wrap header's content in Container too, so it's content would align with content of pages

Isn't the header flex box the container? They're already align through the `alignItems` and the percentage of width that they take

Reply:
I mean aligning the header content with page content verticly:
<img width="648" alt="image" src="https://user-images.githubusercontent.com/32075241/218179996-7beeb3a4-d2ff-4652-aba8-7eff90494021.png">

Reply:
I'm a bit confused as to what you mean sorry...

Reply:
No problem, I will try to explain it deeply.

### Issue 
In my opinion, the content of the header (so custom domains button, certificates etc) should be aligned with the content of the pages that we gonna have). 

Right now it is not aligned:
<img width="648" alt="image" src="https://user-images.githubusercontent.com/32075241/218181192-c9f9e399-72e0-42d1-a62b-f17e08ed8f59.png">
You could see that content starts before the yellow line

### What I propose
Wrap both inner contents for header content of all pages in a container with the same `maxW` (probably `container.xl`, so they would be aligned with each other with all possible screen width

Reply:
https://chakra-ui.com/docs/components/container/usage#container-size

Reply:
Ohh... so you want to remove the spacing on the left to make it more align with the page? Or at least combine the two so they'll be aligned together, am I understanding it right?

Reply:
I don't say that they should be in the same container, we should just assign their containers with the same maxWidth properties and center them in their containers

Reply:
Isn't the header already aligned then? It is the pages that are not aligned with the header?